great_expectations/expectations/expectation.py

```diff
@@ -246,6 +256,43 @@ def wrapper(
     return _param_method
 
 
+def _map_operator_string_to_enum(op_str: str) -> Operator:
+    mapping = {
+        "==": Operator.EQUAL,
+        "!=": Operator.NOT_EQUAL,
+        "<": Operator.LESS_THAN,
+        "<=": Operator.LESS_THAN_OR_EQUAL,
+        ">": Operator.GREATER_THAN,
+        ">=": Operator.GREATER_THAN_OR_EQUAL,
+    }
+    return mapping[op_str]
+
+
+def _convert_string_to_condition(row_condition: str) -> Condition:
+    """Convert legacy string row_condition to new Condition object."""
+    parsed = _parse_great_expectations_condition(row_condition)
+    col = Column(name=str(parsed["column"]))
+
+    if "notnull" in parsed and parsed["notnull"] is True:
```

Comment:
we probably need something similar for the `is_null=True` version of this right?